.circleci/src/workflows/promotion_backend_administration.yml

```
@@ -1,6 +1,5 @@
 when: << pipeline.parameters.run_promotion_backend_administration >>
 jobs:
-  - promote_to_server:
-      name: promote_backend_administration
+  - promote_server:
       context:
         - credentials-ehrenamtskarte
```

🙃 it wants a new line 😀
```suggestion
        - credentials-ehrenamtskarte
        
```

---

Will do in a separate PR :)